styles/style-main.css

```diff
@@ -0,0 +1,95 @@
+.navbar {
```

Comment:
- [ ] It'd be better if we followed Figma's given design, I feel that perhaps you got confused, as you created the base for the desktop version
Figma mobile design:
![image](https://user-images.githubusercontent.com/73140461/143499544-50939605-6167-4f7a-aaf8-4abf88ca3864.png)

Yours:
![image](https://user-images.githubusercontent.com/73140461/143501023-5d06055e-1bf1-4f12-857b-8420a587fef4.png)

Just in case you don't know it already, you can change your browser to view as mobile, by right-clicking the page, clicking on inspect, and clicking here: 
![image](https://user-images.githubusercontent.com/73140461/143499654-c0cbd462-7b18-4141-a76f-7a5093a3f876.png)

And then changing up which size do you want here:
![image](https://user-images.githubusercontent.com/73140461/143499684-7e48a6e5-bdd1-4742-8587-1c42d1726375.png)

Reply:
- [ ] When you're now copying the mobile version, please don't forget to add this as a background and position it correctly:
![image](https://user-images.githubusercontent.com/73140461/143500220-c76ccdbc-0538-4eee-b4a5-80f33bc3f6a7.png)